config/routes.rb

```diff
@@ -6,9 +6,17 @@
 
   resources :episodes do
     resources :announcements
+    resources :timecodes, only: [:create]
   end
 
   get '/admin', to: 'dashboard#index'
   post '/admin/telegram', to: 'dashboard#telegram'
+
   get '*path', to: redirect('/')
+
+  post '/update_youtube_info/:id', to: 'episodes#update_youtube_info'
+
+  post '/add_start/:id', to: 'episodes#add_start'
```

Comment:
what kind of resource is `add_start` ?

Reply:
Changed route to more self explanatory one.